Projects/NeonExample-iOS/ViewController.swift

```diff
@@ -19,25 +19,32 @@ final class ViewController: UIViewController {
 		let boldFont = UIFont.monospacedSystemFont(ofSize: 16, weight: .bold)
 		let italicFont = regularFont.fontDescriptor.withSymbolicTraits(.traitItalic).map { UIFont(descriptor: $0, size: 16) } ?? regularFont
 
+		// Set the default styles. This is applied by stock `NSTextStorage`s during
+		// so-called "attribute fixing" when you type, and we emulate that as
+		// part of the highlighting process in `TextViewSystemInterface`.
+		textView.font = regularFont
+		textView.textColor = .darkGray
+
```

Comment:
From here, this is simpler than injecting the default attributes dictionary into the `TextViewSystemInterface`. IMHO this represents the user's intention best when they set that at all. But it's also a bit weird.